ext/mysqli/tests/bug28817.phpt

```diff
@@ -38,5 +38,7 @@ array(2) {
   [1]=>
   %s(3) "bar"
 }
+
+Warning: main(): Couldn't fetch my_mysql in %s on line %d
```

Comment:
Hm, where does this warning come from?

Reply:
there is a `var_dump($mysql->errno);` on line 25 from where I removed a `@` (it used to be `@var_dump($mysql->errno);`).